src/RF24Logger.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * RF24Logger.h
+ *
+ *  Created on: 2 paź 2020
+ *      Author: wmarkowski
+ */
+
+#ifndef SRC_RF24LOGGER_H_
+#define SRC_RF24LOGGER_H_
+
+#include <WString.h>
+#include "RF24LogLevel.h"
+#include "RF24LogAppender.h"
+
+#define RF24_LOG_LEVEL_ERROR 0
+#define RF24_LOG_LEVEL_WARN 1
+#define RF24_LOG_LEVEL_INFO 2
+#define RF24_LOG_LEVEL_DEBUG 3
+#define RF24_LOG_LEVEL_TRACE 4

```

Comment:
If this is a C++ library primarily, I guess it could just be 
`constexpr uint8_least_t RF24_LOG_LEVEL_TRACE = 4;`. This will add type checking but is as efficient as a define.

Reply:
Indeed, Log levels are defined in RF24LogLevel.h.